src/redux/actions/adminActions.js

```diff
@@ -0,0 +1,57 @@
+import {FETCH_CATEGORY,DELETE_CATEGORY,ADD_CATEGORY,UPDATE_CATEGORY} from './type';
+import axios from 'axios';
+import {url} from '../../helper/url';
+import { tokenConfig } from './authActions';
+
+export const fetchCategory =()=>{
+    return dispatch =>{
+        axios.get(`${url}/categories`)
+            .then(res=>{
+                dispatch({
+                    type:FETCH_CATEGORY,
+                    payload: res.data.categories,
+                })
+            })
+    }
+};
+
+export const deleteCategory =(e)=> (dispatch,getState) =>{
+    const config = tokenConfig(getState);  
```

Comment:
No need to create a new variable config. directly use `tokenConfig(getState)` in the axios call